litebird_sim/coordinates.py

```diff
@@ -1,9 +1,115 @@
 # -*- encoding: utf-8 -*-
+import numpy as np
 
 from astropy.coordinates import BarycentricMeanEcliptic
+from astropy.coordinates import SkyCoord
 
 """The coordinate system used by the framework"""
 DEFAULT_COORDINATE_SYSTEM = BarycentricMeanEcliptic()
 
 """The time scale used by the framework"""
 DEFAULT_TIME_SCALE = "tdb"
+
+e2g = (
```

Comment:
As it is a constant, it should use uppercase letters. Moreover, given that it might be used elsewhere, better to use a more descriptive name, like `ECL_TO_GAL_ROT_MATRIX`